bucket/oracle-instant-client.json

```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://www.oracle.com/database/technologies/instant-client.html",
+    "description": "Oracle Instant Client enables applications to connect to a local or remote Oracle Database for development and production deployment.",
+    "license": "https://www.oracle.com/downloads/licenses/instant-client-lic.html",
+    "version": "19.3.0.0.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-windows.x64-19.3.0.0.0dbru.zip",
+            "hash": "ffceffb0e0024623cbcf5e48409c9c43c36bd117052451cce09faf73325d9931"
+        },
+        "32bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-nt-19.3.0.0.0dbru.zip",
+            "hash": "05d608ce4ee67a3285c4fd4234fb3cfbd5391c48634b10128bea9d8c36cf8456"
+        }
+    },
+    "suggest": {
+       "Microsoft Visual Studio 2017 Redistributable": "vcredist2017"
+    },
+    "extract_dir": "instantclient_19_3",
+    "env_add_path": ".",
+    "pre_install": "mkdir $dir\\network\\admin",
```

Comment:
Why do you precreate this folder? Folder are created automatically on persist

Reply:
That's odd. When I was testing it, it wasn't creating the directory. Perhaps I wasn't on the latest version of `scoop`, though I'm fairly certain I had just updated it.

Reply:
```powershell
> scoop -V
Current Scoop version:
a9fa775d (HEAD -> master, origin/master, origin/HEAD) chore(buckets.json): Update scoop-nonportable URL (#3776)

'main' bucket:
9cc4ca4b (HEAD -> master, origin/master, origin/HEAD) starship: Update to version 0.27.0
```
Removing the `pre-install` directive, I'm not seeing `network/admin` in `$env:SCOOP/apps/oracle-instant-client\current`. Additionally I get the following warning:
```powershell
> scoop install .\oracle-instant-client.json
Installing 'oracle-instant-client' (19.3.0.0.0) [64bit]
Loading instantclient-basic-windows.x64-19.3.0.0.0dbru.zip from cache
Checking hash of instantclient-basic-windows.x64-19.3.0.0.0dbru.zip ... ok.
Extracting instantclient-basic-windows.x64-19.3.0.0.0dbru.zip ... done.
Linking $env:SCOOP\apps\oracle-instant-client\current => $env:SCOOP\apps\oracle-instant-client\19.3.0.0.0
Persisting network\admin
The system cannot find the path specified.
Path not found - D:\scoop\apps\oracle-instant-client\current\network
'oracle-instant-client' (19.3.0.0.0) was installed successfully!
'oracle-instant-client' suggests installing 'vcredist2017'.
```

Reply:
```suggestion
    "pre_install": "if (-not (Test-Path \"$persist_dir\\network\\admin\")) { New-Item \"$dir\\network\\admin\" -ItemType Directory | Out-Null }",
```

Reply:
That still throws the error if the `persist` directory exists on `scoop install oracle-instant-client`:
```
Persisting network\admin
The system cannot find the path specified.
```
This could occur if you install it, then uninstall it, then reinstall it. Could we instead change it to:
```json
    "pre_install": "if (-not (Test-Path \"$dir\\network\\admin\")) { New-Item \"$dir\\network\\admin\" -ItemType Directory | Out-Null }",
```